src/main.go

```diff
@@ -35,6 +35,7 @@ func main() {
 
 	config = loadConfigOrDefault(configFilePath)
 	log.Infoln("Configuration loaded: ", config)
+	defer os.RemoveAll(*config.TemporaryWorkerDirectory)
```

Comment:
I am not sure about this, but I noticed that we do not remove the `temporaryWorkerDirectory` (and any remaining content), what do you think about it @r0x0d ?

Reply:
I think there are a few things for us to answer here before saying `yes` or `no`:

1. Do we want to remove the directory even if it is user specified?
2. The contents inside that folder are deleted at the end of the execution, right? So I believe that there won't be anything inside that folder (for now)?

I'm in favor of deleting the temporary directory, as it is really what it proposes, a temporary directory.

Reply:
Yeah I think it was created with a intention to be only temporary, and you are right that it should be empty, so this should just take care of removing the folder itself

Reply:
Yeah, if there is nothing else in that folder after we finish the script execution, I'm in favor of removing it. Otherwise, I think we should leave the folder there.